status/statusgo_backend_new/core.nim

```diff
@@ -38,4 +38,8 @@ proc callPrivateRPC*(methodName: string, payload = %* []): RpcResponse[JsonNode]
     error "error doing rpc request", methodName = methodName, exception=e.msg
     raise newException(RpcException, e.msg)
     
+proc signMessage*(rpcParams: string): string =
+  return $status_go.signMessage(rpcParams)
 
+proc signTypedData*(data: string, address: string, password: string): string =
+  return $status_go.signTypedData(data, address, password)
```

Comment:
Why are you adding those functions? I cannot see any usage of them?

Reply:
I call it here: https://github.com/status-im/status-desktop/pull/3923/files#diff-ba3e7253c647e8dbc3c6935dc612b99df13b019ffb8432e5b96b624df1fda03eR215